prusti-viper/src/encoder/mir/types/encoder.rs

```diff
@@ -99,7 +105,7 @@ impl<'p, 'v, 'r: 'v, 'tcx: 'v> TypeEncoder<'p, 'v, 'tcx> {
             }
 
             ty::TyKind::Ref(region, ty, _) => {
-                let lft_name = format!("{}", region);
+                let lft_name = encode_lifetime_name(format!("{}", region));
```

Comment:
Maybe there should be a function from `RegionKind` to `String`? I expect many of the `RegionKind` variants will not be visible to us anyway.